src/api/java/com/minecolonies/api/colony/buildings/modules/IDefinesBuildingModule.java

```diff
@@ -0,0 +1,23 @@
+package com.minecolonies.api.colony.buildings.modules;
+
+/**
+ * Interface describing the different modules.
+ */
+public interface IDefinesBuildingModule
```

Comment:
Javadoc completely misleading. And the name is wrong: `IDefinesCoreBuildingStatsModule` is more accurate.